tutorials/gcs-cf-auth/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Google Cloud Functions Access Control
+description: Learn how to add authentication and authorization to Google Cloud Functions using Oauth tokens.
+author: ianmaddox
+tags: serverless, cloud functions, javascript, iam, authn, authz, authorization, authentication
+date_published: 2019-01-08
+---
+# Google Cloud Functions Access Control
+This code demonstrates using Oauth 2.0 authorization tokens in order to control access to a Google Cloud Function. It takes an auth token as a POST parameter and validates that against an IAM controlled GCS bucket before allowing access to the rest of the function. This approach allows service accounts and other authenticated users access to a public HTTPS GCF endpoint without relying on the permissions granted to the default service account attached to all of the Cloud Functions within the GCP project.
+
+# Configuration
+Before the code can be deployed, some customizations must be made. In particular, the GCS bucket to be used for auth must be set. See below for instructions regarding the bucket.
+
+See the configuration file config/default.json for all available options. Best practice is to copy config/default.json to config/local.json and make edits there. More functionality is available for environment-specific configs. See https://www.npmjs.com/package/config for more info.
+
+# Deploying to Cloud Functions
+The exported function name is "example_auth". To deploy through the web UI, open the GCP Cloud Console and then open the Cloud Shell. Cloud Shell can be opened with the ">_" icon in the top-right of the console.
```

Comment:
We typically just say 'Cloud Shell' without the article 'the'.

Consider linking from this first mention to this page:
https://cloud.google.com/shell/

Rather than showing a plain-text approximation of the Activate Cloud Shell icon, you can just give the button's name; that, together with the guidance that you've already given about where the button is, is enough.